### ISSUE TITLE
Uncaught failure of transcode command

### ISSUE THREAD

**Issue by RogueOneEcho** [RogueOneEcho/caesura#78]
## tl;dr;

Docker container inexplicably stops while transcoding RED 1830779
```
exited with code 137
```

Tracks 1 and 2 appear to be 24-bit FLAC with 48000 sample rate
Track 3-19 appear to be 16-bit FLAC with 44100 sample rate

No listing on discogs or musicbrainz

Track durations are also off significantly.

Expected:
Track 1: 01:17:35
Track 2: 01:12:40

Actual:
Track 1: 46:16:32.00
Track 2: 124:44:07.38

## Conclusion
-  ~~verify should have caught the mixed 24-bit and 16-bit during verify~~
-  ~~verify should have caught the mixed 48000 and 44100 sample rates~~
- [x]  verify should have caught the **bit rate**: At least one file has a lower than 192kbps bit rate (1kpbs)

---

## Diagnosis

--- 

Docker container inexplicably stops while transcoding RED 1830779
```
exited with code 137
```
12 CPUs
12 GB RAM

---

16 CPUs
32 GB RAM

```
cargo run -- transcode 1830779 --target v0 
```

![image](https://github.com/user-attachments/assets/e1ca6b68-eecd-4ad1-94f8-813f40dd9614)

Seems to get stuck on one track:

```
cargo run -- transcode 1830779 --target v0                                              
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 0.09s
     Running `target/debug/caesura transcode 1830779 --target v0`
   0.420 INFO ○ Transcoding to V0 for ***
   0.002 INFO ○ [#####################################>--]  2m  18/19  8s remain                                                      thread 'tokio-runtime-worker' panicked at src/transcode/transcode_job.rs:49:22:
Should be able to write to std in: Os { code: 32, kind: BrokenPipe, message: "Broken pipe" }
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
 510.226 ERRO ! Failed to executing task
task 32 panicked with message "Should be able to write to std in: Os { code: 32, kind: BrokenPipe, message: \"Broken pipe\" }"
```

---

Track 1 seems to be problematic?

sox succeeds (eventually) but lame just keeps going

```
sox "01 - (Iteration #001).flac" -G -b 16 -t wav - rate -v -L 48000 dither | lame -S -V 0 --vbr-new --ignore-tag-errors - output.mp3
```
---
All ok with:
```
flac -t
```
---

```
$ ffprobe -i "01 - (Iteration #001).flac" -show_entries format=duration -v quiet -of csv="p=0" 
166592.000000
$ ffprobe -i "02 - (Iteration #002).flac" -show_entries format=duration -v quiet -of csv="p=0" 
449047.381333                                                                                                         
$ ffprobe -i "03 - (Iteration #003).flac" -show_entries format=duration -v quiet -of csv="p=0"
4390.784376
```
---
Running directly /appeared/ to succeed after 5m 46s but output.mp3 is a suspicious 1GB.

```
sox "01 - (Iteration #001).flac" -G -b 16 -t wav - rate -v -L 48000 dither | lame -S -V 0 --vbr-new --ignore-tag-errors - output.mp3
LAME 3.100 64bits (http://lame.sf.net)
polyphase lowpass filter disabled
Encoding <stdin> to output.mp3
Encoding as 48 kHz single-ch MPEG-1 Layer III VBR(q=0)
✔ PIPE|0     5m 46s
```
---

Commands for each track look fine:
```
sox input.flac -G -b 16 -t wav - rate -v -L 48000 dither | lame -S -V 0 --vbr-new --ignore-tag-errors - output.mp3
sox input.flac -G -b 16 -t wav - rate -v -L 48000 dither | lame -S -V 0 --vbr-new --ignore-tag-errors - output.mp3
flac -dcs -- input.flac | lame -S -V 0 --vbr-new --ignore-tag-errors - output.mp3
flac -dcs -- input.flac | lame -S -V 0 --vbr-new --ignore-tag-errors - output.mp3
flac -dcs -- input.flac | lame -S -V 0 --vbr-new --ignore-tag-errors - output.mp3
flac -dcs -- input.flac | lame -S -V 0 --vbr-new --ignore-tag-errors - output.mp3
flac -dcs -- input.flac | lame -S -V 0 --vbr-new --ignore-tag-errors - output.mp3
flac -dcs -- input.flac | lame -S -V 0 --vbr-new --ignore-tag-errors - output.mp3
flac -dcs -- input.flac | lame -S -V 0 --vbr-new --ignore-tag-errors - output.mp3
flac -dcs -- input.flac | lame -S -V 0 --vbr-new --ignore-tag-errors - output.mp3
flac -dcs -- input.flac | lame -S -V 0 --vbr-new --ignore-tag-errors - output.mp3
flac -dcs -- input.flac | lame -S -V 0 --vbr-new --ignore-tag-errors - output.mp3
flac -dcs -- input.flac | lame -S -V 0 --vbr-new --ignore-tag-errors - output.mp3
flac -dcs -- input.flac | lame -S -V 0 --vbr-new --ignore-tag-errors - output.mp3
flac -dcs -- input.flac | lame -S -V 0 --vbr-new --ignore-tag-errors - output.mp3
flac -dcs -- input.flac | lame -S -V 0 --vbr-new --ignore-tag-errors - output.mp3
flac -dcs -- input.flac | lame -S -V 0 --vbr-new --ignore-tag-errors - output.mp3
flac -dcs -- input.flac | lame -S -V 0 --vbr-new --ignore-tag-errors - output.mp3
flac -dcs -- input.flac | lame -S -V 0 --vbr-new --ignore-tag-errors - output.mp3
```

---

Proplis report:

**At least one file has a lower than 192kbps bit rate (1kpbs).**

```
⮕ Analysing content/Brian Grainger - Cursive Bell (2018)
▻ Checking Path is a music release
  🗹  | 2.3.1      | Release contains FLAC files
  🛈  |  --        | Total size of release folder: 2120.01Mb.
▻ Checking music files
  🞎  | 2.1.6      | Could not confirm the same encoder was used: found different vendor strings
  🞎  | 2.1.6      | The tracks do not have the same bit depth.
  🞎  | 2.1.6.2    | ⮕ At least one track is 24bit FLAC when the rest is 16bit, acceptable for some WEB releases.
  🞎  | 2.1.6      | Release has a mix of sample rates, acceptable for some WEB releases (2.1.6.2).
  🗹  | 2.1.1      | ⮕ All sample rates are less than or equal to 192kHz.
  🗷  | 2.1.3      | At least one file has a lower than 192kbps bit rate (1kpbs).
  🗹  | 2.1.6.3    | Release does not also contain other kinds of music files.
  🗹  | 2.2.10.8   | Integrity checks successful for all FLACs, no ID3 tags detected.
  🗹  |  --        | No ID3v1 tags detected in the first track.
  🗹  | 2.2.10.10  | First track is not uncompressed FLAC.
  🗹  | upload#DNU | The first track does not contain MQA encoder metadata.
  🗹  | upload#DNU | The first track does not contain MQA syncwords.
  🗹  |  --        | The first track does not contain padded bits.
▻ Checking organization
  🗹  | 2.3.12     | Maximum character length is less than 180 characters.
  🗹  |  --        | No non-standard unicode spaces detected.
  🗹  | wiki#371   | Release only contains allowed extensions.
  🗹  | 2.3.3      | Release does not have empty folders or unnecessary nested folders.
  🗹  | 2.3.20     | No leading space/dot found in files and folders.
  🗷  | 2.3.15     | Tracks seem incorrectly organized: track number is not a number
▻ Checking tags
  🗷  | 2.3.16.1/4 | At least one tracks is missing required tags.
  🗷  | 2.3.16.1/4 | ⮕ Error: content/Brian Grainger - Cursive Bell (2018)/01 - (Iteration #001).flac: missing minimal tag(s) ARTIST, TITLE, ALBUM, TRACKNUMBER
  🗷  | 2.3.16.1/4 | ⮕ Error: content/Brian Grainger - Cursive Bell (2018)/02 - (Iteration #002).flac: missing minimal tag(s) ARTIST, TITLE, ALBUM, TRACKNUMBER
  🗹  | 2.3.19     | All tracks have embedded art + padding blocks of a total size smaller then 1024 KiB.
  🗷  |  --        | Tracks have inconsistent tags about the release: inconsistent album titles, inconsistent album artists
  🗹  |  --        | Artist/Album artist tags seem consistent.
  🗹  | 2.3.18.3   | Track number for the first track is not combined with track total.
  🗷  | 2.1.19     | Checking for missing files: cannot check with A1, A2, ..., B1, B2, ... vinyl numbering
▻ Checking filenames
  🗹  |  --        | Tracks filenames do not appear to contain problematic characters.
  🗹  |  --        | Track filenames have lower case extensions.
  🗷  | 2.3.13     | At least one track filename does not contain its track number.
  🗹  | 2.3.11     | All tracks filenames appear to contain at least the beginning of song titles.
  🗷  |  --        | Could not check filename/subfolder order. Track/Disc numbers might not be numbers: track number is not a number
▻ Checking extra files
  🞎  |  --        | Cannot find cover.jpg in top folder or in all disc subfolders, consider adding one or renaming the cover to that name.
  🗹  |  --        | Release has 1 accompanying files.
  🛈  |  --        | Total size of accompanying files: 0.09Mb.
  🗹  |  --        | Accompanying files represent 0.00% of the total size.
▻ Checking folder name
  🗹  | 2.3.2      | Title of album is in folder name.
  🗹  | 2.3.2      | All album artists found in folder name.
  🞎  | 2.3.2      | Format (FLAC) not found in folder name.
  🞎  | 2.3.2      | Since release seems to contain 24bit FLACs, the folder name could mention it.
  🞎  | 2.3.2      | Since release does not .log/.cue, it is probably a WEB or Vinyl release. The folder name could mention it.

▻ Results
24 checks OK, 9 checks KO, and 8 warnings.
```

**Comment by RogueOneEcho** [comment]
Should be fixed in 6c1e63cf
